catalyst/utils/hydra_config.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from omegaconf import DictConfig, OmegaConf
+
+
+def prepare_config(cfg: DictConfig) -> DictConfig:
```

Comment:
maybe, `prepare_hydra_config`? and resolve naming?

Reply:
ok